examples/dynamic_routing/envoy_config.json

```diff
@@ -306,7 +306,7 @@
                             "route": {
                               "cluster": "http-bookstore-edf123456-uc.a.run.app:443",
                               "hostRewriteLiteral": "http-bookstore-edf123456-uc.a.run.app",
-                              "timeout": "30s"
```

Comment:
These are timeouts to the backend. I changed the values to make them unique, they're unrelated to the `--http_request_timeout_s`.